sentry/src/main/java/io/sentry/event/interfaces/SentryException.java

```diff
@@ -30,6 +31,24 @@
      * @param childExceptionStackTrace StackTrace of the exception caused by {@code throwable}.
      */
     public SentryException(Throwable throwable, StackTraceElement[] childExceptionStackTrace) {
+        this(throwable, childExceptionStackTrace, null);
+    }
+
+    /**
+     * Creates a Sentry exception based on a Java Throwable.
+     * <p>
+     * The {@code childExceptionStackTrace} parameter is used to define the common frames with the child exception
+     * (Exception caused by {@code throwable}).
+     *
+     * @param throwable                Java exception to send to Sentry.
+     * @param childExceptionStackTrace StackTrace of the exception caused by {@code throwable}.
+     * @param exceptionMechanism The {@link ExceptionMechanism} of the {@code throwable}.
```

Comment:
I think the Javadoc should clearly state that this parameter is nullable or optional.